### DISCUSSION TITLE
Apollo Server 3 and TypeGraphql

### DISCUSSION THREAD

**Discussion by edw19** [MichalLytek/type-graphql#1011] [Q&A] [answered]
I am thinking of starting a new one and I have decided to use the new apollo server 3 with typegraphql, in the apollo server launch information they say that they have removed support for subscriptions, how much does this affect typegraphql and its support for subscriptions 🤔🤔🤔🤔🤔🤔🤔

**Comment by MichalLytek** [comment, marked as answer]
TypeGraphQL, Nexus, `graphql-js` -> builds a `GraphQLSchema` object

Apollo Server, GraphQL Helix, GraphQL Yoga -> takes `GraphQLSchema` and expose that via HTTP or websockets

So answering your question: it completly does not affect TypeGraphQL at all. You just need to create your GraphQL server in a new way.
